### PR TITLE
Update how-windows-update-works.md

### PR DESCRIPTION
**Pull request by itsrlyAria**
Adding Proxy behavior information as these changes were added last year to all versions of Windows 10+.

### CODE REVIEW

File: windows/deployment/update/how-windows-update-works.md [hunk 1]
```diff
@@ -90,6 +90,15 @@ When users start scanning in Windows Update through the Settings panel, the foll
    - Windows Update uses the thread ID filtering to concentrate on one particular task. 
 
       ![Windows Update scan log 1.](images/update-scan-log-1.png)
+      
+#### Proxy Behavior
+For Windows Update (WU) scans URLs which are used for update detection ([MS-WUSP]: SimpleAuth Web Service | Microsoft Docs, [MS-WUSP]: Client Web Service | Microsoft Docs):
```
Comment (JohanFreelancer9) [inline windows/deployment/update/how-windows-update-works.md:95]:
```suggestion
For Windows Update (WU) scans URLs that are used for update detection ([MS-WUSP]: SimpleAuth Web Service | Microsoft Docs, [MS-WUSP]: Client Web Service | Microsoft Docs:
```

File: windows/deployment/update/how-windows-update-works.md [hunk 2]
```diff
@@ -90,6 +90,15 @@ When users start scanning in Windows Update through the Settings panel, the foll
...
+- System proxy is attempted (set using the netsh command).
```
Comment (JohanFreelancer9) [inline windows/deployment/update/how-windows-update-works.md:96]:
```suggestion
- System proxy is attempted (set using the Netsh command).
```

File: windows/deployment/update/how-windows-update-works.md [hunk 3]
```diff
@@ -90,6 +90,15 @@ When users start scanning in Windows Update through the Settings panel, the foll
...
+- If WUA fails to reach the service due to a certain proxy, service, or authentication error code, then user proxy is attempted (generally it is the logged-in user) Note: For intranet WSUS update service URLs, we provide an option via Windows Update policy to select the proxy behavior.
```
Comment (JohanFreelancer9) [inline windows/deployment/update/how-windows-update-works.md:97]:
```suggestion
- If WUA fails to reach the service due to a certain proxy, service, or authentication error code, then user proxy is attempted (generally it is the logged-in user). **Note**: For intranet WSUS update service URLs, we provide an option via Windows Update policy to select the proxy behavior.
```

File: windows/deployment/update/how-windows-update-works.md [hunk 4]
```diff
@@ -90,6 +90,15 @@ When users start scanning in Windows Update through the Settings panel, the foll
...
+
+For WU URLs which are NOT used for update detection (download, reporting etc):
```
Comment (JohanFreelancer9) [inline windows/deployment/update/how-windows-update-works.md:99]:
```suggestion
For WU URLs that are *not* used for update detection (such as download and reporting):
```

File: windows/deployment/update/how-windows-update-works.md [hunk 5]
```diff
@@ -90,6 +90,15 @@ When users start scanning in Windows Update through the Settings panel, the foll
...
+- User proxy is attempted.
+- If WUA fails to reach the service due to certain proxy, service, or authentication error code, then the system proxy is attempted.
```
Comment (JohanFreelancer9) [inline windows/deployment/update/how-windows-update-works.md:101]:
```suggestion
- If WUA fails to reach the service due to a certain proxy, service, or authentication error code, then the system proxy is attempted.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @itsrlyAria, kindly commit the suggested changes. Many thanks.

File: windows/deployment/update/how-windows-update-works.md (hunk 1 above)
Comment (aczechowski) [reply windows/deployment/update/how-windows-update-works.md:95]: I fixed this term, but kept the end parentheses which is needed

File: windows/deployment/update/how-windows-update-works.md (hunk 2 above)
Comment (aczechowski) [reply windows/deployment/update/how-windows-update-works.md:96]: Arguably, the command is `netsh` from the command prompt. The usage shows all lowercase. So I wrapped it in \` 😄

File: windows/deployment/update/how-windows-update-works.md (hunk 3 above)
Comment (aczechowski) [reply windows/deployment/update/how-windows-update-works.md:97]: I fixed this to be a proper `> [!NOTE]`

File: windows/deployment/update/how-windows-update-works.md (hunk 4 above)
Comment (aczechowski) [reply windows/deployment/update/how-windows-update-works.md:99]: I fixed this, mostly by using contraction "_aren't_" and with emphasis

**Review by aczechowski — APPROVED** [review]
Thanks @itsrlyAria for the contribution, and thank you for what appears to be your first to the Windows repository! 🎉 I made some tweaks based on feedback from @JohanFreelancer9, but otherwise looks good. The changes will publish live around 11 AM Pacific on 4 May 2022.

**Pull request merged**